pkg/internal/mysqlcluster/mysqlcluster.go

```diff
@@ -170,19 +167,37 @@ func (c *MysqlCluster) GetMasterHost() string {
 	return masterHost
 }
 
+// GetMySQLSemVer returns the MySQL server version in semver format, or the default one
+func (c *MysqlCluster) GetMySQLSemVer() semver.Version {
+	version := c.Spec.MysqlVersion
+	// lookup for an alias, usually this will solve 5.7 to 5.7.x
+	if v, ok := constants.MySQLTagsToSemVer[version]; ok {
+		version = v
+	}
+
+	sv, err := semver.Make(version)
+	if err != nil {
```

Comment:
This is potentially dangerous. If the user specifies an invalid semver tag, for example, `5.7.13-build.3`, we would override their desire to run MySQL `5.7.13` with our default `5.7.26`.

Reply:
done, now returns `0.0.0` which will not have a related image